### PR TITLE
Added "--capabilities" command and make version command fail when there is an error

### PR DESCRIPTION
**Pull request by richardpen**
Tested with simple code:
```
./bin/plugins/ecs-ipam --version
{"version":"2017.06.0","dirty":true,"gitShortHash":"226db36"}
 ./bin/plugins/ecs-ipam --capabilities
{"capabilities":["awsvpc-network-mode"]}
```

### CODE REVIEW

File: plugins/ecs-bridge/main.go [hunk 1]
```diff
@@ -47,8 +47,7 @@ func main() {
 
 	if printVersion {
 		if err := printVersionInfo(); err != nil {
-			os.Stderr.WriteString(
-				fmt.Sprintf("Error getting version string: %s", err.Error()))
+			fmt.Printf("Error getting version string: %s", err.Error())
```
Comment (vsiddharth) [inline plugins/ecs-bridge/main.go]: `Printf` writes to `stdout` while the original wrote to `stderr`. 
Is this intentional ?

Comment (richardpen) [inline plugins/ecs-bridge/main.go]: yes, write to the stderr won't be visible to the [err](https://github.com/aws/amazon-ecs-agent/blob/enis/agent/ecscni/plugin.go#L185). Write to the stdout will makes it captured by the err.

Comment (samuelkarp) [inline plugins/ecs-bridge/main.go]: You can definitely retrieve `stderr` from processes launched by Go.  I'd prefer to have errors on `stderr`.

Comment (aaithal) [inline plugins/ecs-bridge/main.go]: Would prefer to revert to `os.Stderr` here.

File: plugins/eni/main.go [hunk 2]
```diff
@@ -47,10 +47,9 @@ func main() {
 
 	if printVersion {
 		if err := printVersionInfo(); err != nil {
-			os.Stderr.WriteString(
-				fmt.Sprintf("Error getting version string: %s", err.Error()))
+			fmt.Printf("Error getting version string: %s", err.Error())
```
Comment (aaithal) [inline plugins/eni/main.go]: Same as above. Can you revert to using `os.Stderr.WriteString` ?

File: plugins/ipam/main.go [hunk 3]
```diff
@@ -40,10 +40,9 @@ func main() {
 
 	if printVersion {
 		if err := printVersionInfo(); err != nil {
-			os.Stderr.WriteString(
-				fmt.Sprintf("Error getting version string: %s", err.Error()))
+			fmt.Printf("Error getting version string: %s", err.Error())
```
Comment (aaithal) [inline plugins/ipam/main.go]: Same as above. Can you revert to using `os.Stderr.WriteString` ?

**Pull request merged**